MigrationCodelab/app/src/main/java/com/google/samples/apps/sunflower/viewmodels/PlantDetailViewModel.kt

```diff
@@ -36,7 +37,7 @@ class PlantDetailViewModel(
     val plant = plantRepository.getPlant(plantId)
 
     fun addPlantToGarden() {
-        viewModelScope.launch {
+        viewModelScope.launch(Dispatchers.IO) {
```

Comment:
I'd suggest using `withContext(Dispatchers.IO)` inside the `createGardenPlanting` suspend fun, because it's a convention to have safe-main coroutines

Reply:
Good call out. Fixed in 66996ac